src/libs/events.js

```diff
@@ -25,6 +25,7 @@ const eventsList = {
   billingProjectGoToWorkspace: 'billing:project:workspace:navigate',
   billingProjectOpenFromList: 'billing:project:open-from-list',
   billingProjectSelectTab: 'billing:project:tab',
+  billingChangeAccount: 'billing:project:account:update',
```

Comment:
Just renamed keys to this object so that they are all in the "billing" section. The MixPanel event names are unchanged.